vulnerabilities/management/commands/commit_export.py

```diff
@@ -0,0 +1,146 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# VulnerableCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/aboutcode-org/vulnerablecode for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+import logging
+import os
+import shutil
+import tempfile
+from datetime import datetime
+from pathlib import Path
+from urllib.parse import urlparse
+
+import requests
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandError
+from git import Repo
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = """Commit the exported vulnerability and package in backing git repository"""
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "path",
+            help="Path to exported data.",
+        )
+
+    def handle(self, *args, **options):
+        if path := options["path"]:
+            base_path = Path(path)
+
+        if not path or not base_path.is_dir():
+            raise CommandError("Enter a valid directory path")
+
+        export_repo_url = os.environ.get("VULNERABLECODE_EXPORT_REPO_URL", None)
+        github_service_token = os.environ.get("GITHUB_SERVICE_TOKEN", None)
+        github_service_name = os.environ.get("GITHUB_SERVICE_NAME", None)
+        github_service_email = os.environ.get("GITHUB_SERVICE_EMAIL", None)
+
+        local_dir = tempfile.mkdtemp()
+        current_date = datetime.now().strftime("%Y-%m-%d")
+
+        branch_name = f"export-update-{current_date}"
+        commit_message = f"Update package and vulnerability data\nSigned-off-by: {github_service_name} <{github_service_email}>"
```

Comment:
Let's add more details because this is from VulnerableCode, using a specific instance (public) and using a given version of VCIO.
```
"""
Update package vulnerabilities from VulnerableCode

Tool: pkg:github/aboutcode-org/vulnerablecode@v34.223232 (or the purl of whatever tool we run)
Reference: https://public.vulnerablecode.io (as default, otherwise a provided URL) 
Signed-off-by: {github_service_name} <{github_service_email}>
"""
```

Reply:
Done